### ISSUE TITLE
Fix stdout+stderr redirection in OSX

### ISSUE THREAD

**Issue by bcumming** [arbor-sim/nsuite#41]
Replace
```
git checkout "$repo" &>> "$out"
```
with
```
git checkout "$repo" >> "$out" 2>&1
```
Because OSX (BSD) doesn't understand the first form.

**Comment by bcumming** [comment]
Full OSX support is taking longer than hoped, because of the many little differences between Posix implementations on OS X and Linux.
Current status
* **Arbor**: compiles and passes all benchmarks+tests.
* **Neuron**: compiles, but installing the Python bindings doesn't quite work.
* **CoreNeuron**: haven't started. I expect some surprises, because CoreNeuron.

**Comment by halfflat** [comment]
Redirection fixes included in #47.
Neuron issue now #48.
